City-of-Helsinki/drupal-hdbt
# UHF-218 Admin theme adjustments

To test:

1. Setup / go to your local helfi site or make new site with oneliner.
2. Checkout the branch under hdbt-theme
3. make drush-cr
4. Log into the site
5. Make sure that the scroll bar that was caused by the toolbar is now gone from the site.
